### ISSUE TITLE
Incorrectly marked as Count when should be Gauge

### ISSUE THREAD

**Issue by warlockza** [60East/amps-integration-datadog#4]
https://github.com/60East/amps-integration-datadog/blob/6c0b53ab8f913f4a5a628ab575dbb8930591bdc4/amps.py#L67

All the _per_second metrics should be Gauge or rate, not counters, otherwise datadog tries to treat them as counters and smooths them over graphing windows

**Comment by brand-zz** [comment]
Thanks for the note, @warlockza !   Won't the guage/rates be smoothed over a graphing window as well?

**Comment by warlockza** [comment]
No, fortunately not. I override the metric type in datadog and it fixed a
lot of issues I have been having with the scaling.

Mike
